src/main/java/com/epam/jdbc/service/AcademicClassesService.java

```diff
@@ -0,0 +1,6 @@
+package com.epam.jdbc.service;
+
+public interface AcademicClassesService {
+
+    int getIDByAcademicClass(String academicClass);
```

Comment:
Please change the method name to findIDByAcademicClass(), because all get operations are written with  "find".

Reply:
done